api/API.md

```diff
@@ -196,6 +196,105 @@ The `config` endpoint, specifically the `EdgeDevConfig` message, supports arbitr
 
 The `EdgeDevConfig` message can contain zero, one or more `ConfigItem` entries, each of which is an arbitrary key/value pair. These SHOULD be used to send implementation-specific configuration to a Device, other than configuration already defined in this API or the messages. The items are defined in [configuration properties](../docs/CONFIG-PROPERTIES.md)
 
+### Controller Certificates
+
+Retrieve Certificates that Controller would use for signing or any key exchanges. Controller can include one or more certificates in the response, and include information about each certificate such as unique identifier for the certificate, the target usage for the certificate and any other properties related to the certificate. Each certificate MUST be a X.509 certificate in PEM format.
+
+   GET /api/v1/edgeDevice/ControllerCerts
+
+Return codes:
+
+* No such certificates to send: `404`
+* One or more certificates found: `200`
+
+Request:
+
+The request MUST use HTTP for this request
+
+The request MUST NOT contain any body content
+
+Response:
+
+The response mime type MUST be "application/x-proto-binary".
+The response MUST contain a message with list of certificates to use with the Controller. The body MUST be a protobuf message of type [certs.ZControllerCert](./proto/certs/certs.proto).
+
+### Edge Node Certificates
+
+Publish certificates that edge device would use for signing or any key exchanges. Edge node can include one or more certificates in the request, and include information abou
```

Comment:
Please add blank line after first sentence, since that sentence is basically a sub-title to the section.

"about" has a blank before the "t"

isMutable description seems inverted; isMutable is not set you can't change them

Reply:
updated.